testcases.py

```diff
@@ -1336,6 +1332,10 @@ def name():
     def abbreviation():
         return "BA"
 
+    @staticmethod
+    def testname(p: Perspective):
+        return "transfer"
+
```

Comment:
This means that implementation that don't implement path validation yet (like quic-go 🤐) will fail this test. I think this is ok, since this is a mandatory part of the RFC. Might lead to some complaints though if the matrix is getting slightly more red as a result of merging this PR.

Reply:
No progress without pain.